hugo/static/.htaccess

```diff
@@ -77,8 +76,10 @@ RewriteRule ^files/(.*)$ anthology-files/files/$1 [L,NC]
 # Thumbnails (e.g., /thumb/P17-1069.jpg loads anthology-files/thumb/P17-1069.jpg)
 RewriteRule ^thumb/(.*)$ anthology-files/thumb/$1 [L,NC]
 
-# Author pages (e.g., people/arya-d-mccarthy loads people/a/arya-d-mccarthy)
-RewriteRule ^people/([a-z])([\-a-z0-9]*)$ people/$1/$1$2/ [L,NC]
+# Author pages. A request for the old-style URL /people/m/matt-post should give a 301 redirect
+# to /people/matt-post. Note that this works for any letter prefix, but this can't be avoided
+# since the pattern-match can't match source side.
+RewriteRule ^people/[a-z]/([a-z0-9])([\-a-z0-9]+)/?$ people/$1$2/ [R=301,L,NC]
```

Comment:
Minor comment: If the pattern-match can't be used source-side anyway, we don't need two capture groups, do we? This could just be

`RewriteRule ^people/[a-z]/([\-a-z0-9]+)/?$ people/$1/ [R=301,L,NC]`